src/differentials/thunks.jl

```diff
@@ -61,9 +60,6 @@ Do not use `@thunk` if this would be equal or more work than actually evaluating
 - The expression being a constant
 - The expression is merely wrapping something in a `struct`, such as `Adjoint(x)` or `Diagonal(x)`
 - The expression being itself a `thunk`
-- The expression being from another `rrule` or `frule` (it would be `@thunk`ed if required by the defining rule already)
-- There is only one derivative being returned, so from the fact that the user called `frule`/`rrule`
-they clearly will want to use that one.
```

Comment:
Not sure what happened here. Need to add them back in

Reply:
bad rebase?

Reply:
fixed